tests/components/sighthound/test_image_processing.py

```diff
@@ -91,3 +94,19 @@ def capture_person_event(event):
     state = hass.states.get(VALID_ENTITY_ID)
     assert state.state == "2"
     assert len(person_events) == 2
+
+
+async def test_save_image(hass, mock_image, mock_detections):
+    """Save a processed image."""
+    VALID_CONFIG[ip.DOMAIN].update({sh.CONF_SAVE_FILE_FOLDER: TEST_DIR})
```

Comment:
We shouldn't modify this constant dict here. Make a copy first.

Reply:
yup